src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/loadbalancer/ApacheEc2LoadBalancerFormula.kt

```diff
@@ -35,16 +36,19 @@ internal class ApacheEc2LoadBalancerFormula : LoadBalancerFormula {
                 launch
                     .withSecurityGroupIds(httpAccess.groupId)
                     .withSubnetId(subnet.subnetId)
-                    .withInstanceType(InstanceType.M1Large)
+                    .withInstanceType(InstanceType.M5Large)
             }
         )
+        key.file.facilitateSsh(ssh.host.ipAddress)
+        val loadBalancer = HaProxyLoadBalancer(
```

Comment:
If we want users to consume `ApacheEc2LoadBalancerFormula`, it should be moved to the `api` subpackage and we should mention we `### Added` new API in the `CHANGELOG.md`

Reply:
We definitely want this available for users to consume (including internal Jira perf. tests). Let's move to api and adjust the `CHANGELOG.md` accordingly.